### PR TITLE
🐛 Set a destination scope to hasAction method to prevent checking tap presence on parents elements on amp-accordion

### LINKED ISSUE #24118: New amp-accordion headings error?
## What's the issue?

We have begun seeing a new error show up in the console that reads "amp-accordion headings should not have tap actions registered." for every use of the amp-accordion component on a page.

We have not modified this markup for quite some time so we are having difficulty troubleshooting the new error.

In looking at each use of amp-accordion, we also can not identify any tap actions being registered.

## How do we reproduce the issue?

1. Open https://www.google.com/amp/s/www.ebags.com/amp/product/tumi/cfx-bayview-travel-duffel/365912
2. View errors listed in the console.

## What browsers are affected?

This appears to be across browsers and device type.

## Which AMP version is affected?

This appears to be a new issue.
Version 1908162134430

### PR DESCRIPTION
**Pull request by BurakDev**
Closes #24118 
Fix bug introduced in #23720, resolve issue #24118

### CODE REVIEW

**Comment by BurakDev** [comment]
@googlebot I signed it!

File: extensions/amp-accordion/0.1/amp-accordion.js [hunk 1]
```diff
@@ -188,7 +188,7 @@ class AmpAccordion extends AMP.BaseElement {
       this.headers_.push(header);
 
       userAssert(
-        this.action_.hasAction(header, 'tap') == false,
+        this.action_.hasAction(header, 'tap', header.parentElement) == false,
```
Comment (wassgha) [inline extensions/amp-accordion/0.1/amp-accordion.js]: You can either use `section` or `dev().assertElement(header.parentElement)` to satisfy the type check (preference to `section` in my opinion).

**Comment by wassgha** [comment]
@BurakDev just a small tip for the future, if you type `Closes #[issue number]` in your description the issue will be automatically closed as soon as the PR is merged. I edited that for you :)
Also since this is a time sensitive issue, please let me know if you can't make the requested changes and I'll do them!

**Comment by pstricker** [comment]
@BurakDev Thanks for hopping on this issue! We really appreciate it.

**Pull request merged**